kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -466,12 +473,11 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
       podLabelUniqueKey: String): Unit = {
     try {
       val kubernetesClient = getOrCreateKubernetesClient(kubernetesInfo)
-      val deleted = if (podName == null) {
-        !kubernetesClient.pods()
-          .withLabel(LABEL_KYUUBI_UNIQUE_KEY, podLabelUniqueKey)
-          .delete().isEmpty
-      } else {
-        !kubernetesClient.pods().withName(podName).delete().isEmpty
+      val deleted = Option(podName) match {
```

Comment:
No functionality change? TBH, I prefer the original style.

Reply:
reverted